script/Common/00_00.txt

```diff
@@ -2032,17 +2037,17 @@ But that's okay.
 [sha:b3f91ef4d144ecf79e47dbe95c639a17b0884d06]{
 -- Page 154, Offset 495.
 -- 遠野志貴はひとりでもちゃんとやっていける。
-Tohno Shiki can do fine on his own.
+Tohno Shiki would do fine on his own.
 }
 [sha:ff0dc147a5af19ba6eb5eba858865e1653ed8a43]{
 -- Page 155, Offset 496.
 -- 新しい生活を、新しい家族と送る。
-A new life with a new family.
+I could lead a new life with a new family.
 }
 [sha:b14b03923db26e59cc94a418d2d102fb444807be]{
 -- Page 155, Offset 497.
 -- 10歳の夏を過ごした病室を後にする。
-I left the hospital room where I spent my tenth summer.
+I left the hospital room where I had spent my tenth summer.
```

Comment:
Referring to my comment above, it feels a bit like Shiki had paused narration to sprinkle in his own thoughts about his past in hindsight, and is only continuing the narration now. I know it's not in the og text but I can't help but feel this sentence needs a 'And thus,' or something preceding it.

Reply:
Honestly, partially I think it would be easier to almost just make this entire scene present tense. It feels like that's what it should be at times...